.env.docker

```diff
@@ -14,4 +14,4 @@ ADMIN_USERS="eddiejaoude,SaraJaoude"
 STRIPE_SECRET_KEY=""
 STRIPE_PREMIUM_PRICING_ID=""
 STRIPE_WEBHOOK_SECRET=""
-NEXT_PUBLIC_PREMIUM_SUPPORT_URL=""
+NEXT_PUBLIC_PREMIUM_SUPPORT_URL=""
```

Comment:
Not sure what this change is for? Also GitHub marks it as red, please remove these changes